package.json

```diff
@@ -44,7 +44,7 @@
     "@testing-library/react-hooks": "^8.0.1",
     "@types/jest": "^29.1.0",
     "@types/node": "^18.8.2",
-    "@types/react": "^18.0.21",
```

Comment:
Muss des so? 😂

Reply:
![grafik](https://github.com/HASKI-RAK/HASKI-Frontend/assets/106922840/f14b92e6-8050-4d8a-b49d-7adee4e7944c)

Reply:
Das passiert, wenn ich in irgendeiner Hinsicht die Versionen ändere